app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -17,9 +17,6 @@ class Admin::AdminSetsController < ApplicationController
     # Used to get the members for the show action
     self.member_search_builder_class = Hyrax::AdminAdminSetMemberSearchBuilder
 
-    # Used to get a list of admin sets for the index action
-    self.list_search_builder_class = Hyrax::AdminSetSearchBuilder
```

Comment:
@carrickr @darrenleeweber This is dead/unused code, I guess? @jcoyne can you confirm?